autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/mod.rs

```diff
@@ -289,6 +289,8 @@ pub struct ClientFlags {
     pub check_storage: bool,
     /// Flags the need to drop the user record
     pub old_record_version: bool,
+    /// First time a user has connected "today"
+    pub user_gm: bool,
```

Comment:
Just curious about the name `user_gm`. What does the `gm` stand for? Forgive my ignorance if it's a metrics-related acronym I don't know. If not, I'm curious maybe if a more imperative variable name.

Reply:
"good morning". It's fairly meaningless but I needed something to hold the fact that this is the first connection of the day.

I'm very open to better names.

Reply:
With it solely being used for emitting the new metric I suggest something like `emit_channel_metrics`.